lib/tasks/nightly_cleanup.rake

```diff
@@ -3,6 +3,9 @@ task nightly_cleanup: :environment do
   User.all.each { |user| user.clean_call_list_between_shifts }
   puts "#{Time.now} -- cleared all recently reached patients from call lists"
 
+  User.disable_inactive_users
+  puts "#{Time.now} -- locked accounts of users who have not logged in since #{9.months.ago}"
```

Comment:
Suggestion: Replace `#{9.months.ago}` with the `TIME_BEFORE_DISABLED_BY_FUND` so that if it's updated this doesn't get out of date

Reply:
great flag!